### ISSUE TITLE
Windows 7 network issue

### ISSUE THREAD

**Issue by Mokatsu** [cuckoosandbox/cuckoo#2557]
Thanks for creating an issue! But first: did you read our community guidelines?
https://cuckoo.sh/docs/introduction/community.html

##### My issue is:
The issue is with the VM setup maybe someone has seen this.
I install my guest windows 7 vm and set the host-only adapter, then I apply the simple routing (iptables) to give it internet access.

(Quick note) when I do this with an Ubuntu guest my guest has internet access

When do this with the windows 7 guest I get no internet  - what is weird is that i can ping/traceroute fine, but when I try to do an nslookup i get query timeout (on virtualbox) query refused (on kvm)

On kvm when I do a bridge (for the windows 7 guest) i get internet everything works fine
On virtualbox (when I do NAT) windows 7 guest everything works fine.

It only breaks when I do host only (even with proper IPtables
* again this configuration works fine with a ubuntu guest

##### My Cuckoo version and operating system are:
Cuckoo=2.0.6
Host = Ubuntu16.04
Guest = Windows 7 Professional
Virtualization= Tried kvm and virtualbox

##### This can be reproduced by:
install windows 7 and set the adapter to host-only with simple routing iptables
##### The log, error, files etc can be found at:
windows 7 nslookup - query refused (kvm)
windows 7 nslookup - query timed out (virtualbox)

**Comment by doomedraven** [comment]
logs?

**Comment by Mokatsu** [comment]
nslookup results - nslookup www.google.com
DNS request timed out.
server UnKnown
Address 8.8.8.8

ipconfig /all
ipv4 addrs: 192.168.1.120
submask: 255.255.255.0
gateway: 192.168.1.1
DNS servers 8.8.8.8

IPtables on the host
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain FORWARD (policy DROP)
target     prot opt source               destination         
ACCEPT     all  --  anywhere             anywhere             state RELATED,ESTABLISHED
ACCEPT     all  --  192.168.1.0/24       anywhere            
ACCEPT     all  --  192.168.1.0/24       192.168.1.0/24      
LOG        all  --  anywhere             anywhere             LOG level warning

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination

I used the simple routing guide.

Not sure what else to show

**Comment by doomedraven** [comment]
so what have solved the issue?

**Comment by Mokatsu** [comment]
I accidentally closed it.

**Comment by Mokatsu** [comment]
Weird thing not sure if it matter, with the windows machined if run inetsim on the cuckoo host, and go to any http webpage, it redirect me to the fake inetsim page (within the browser)

**Comment by doomedraven** [comment]
then its your iptables, check when analysis starts

**Comment by doomedraven** [comment]
stupid question but, both in the same network? win and linux guests, is networking in both guest configured in the same way?

**Comment by Mokatsu** [comment]
Yes, the only difference is the static machine ip windows is 192.168.1.120 and linux is 192.168.1.121, same dns server, same gateway, same subnet.

Linux works, windows is does not (but windows guest can ping but DNS does not work)

**Comment by doomedraven** [comment]
well then we have part of the problem, now you need to do tcpdump or wireshark and see if dns req forwarded to 8.8.8.8 lets say or dropped on host to know where is the problem

**Comment by Mokatsu** [comment]
I am Able to upload pcaps? or should i do screenshots?

**Comment by doomedraven** [comment]
well you also can check it and say is no? :P

**Comment by Mokatsu** [comment]
On the Window Guest, I see the DNS queries, but no replies,
On the host (eth0) interface (the one that has an internet connection) nothing from the Windows Guest is seen except ping requests.
When I also pulled the traffic from the host, on (vboxnet0) I saw the DNS queries no replies.
To complicate the matter further, when I recorded traffic from my Linux guest, everything seems to be fine, I see DNS queries, and I am able to see the HTTP traffic.

The only difference btw Windows and Linux guests is that the DNS server is unknown on the Windows Guest.

**Comment by Mokatsu** [comment]
This looks like it's an iptable issue but why does it work for Linux guest but not for Windows guest?

**Comment by doomedraven** [comment]
Bcz windows love to make us spend hours investigating lol

**Comment by Mokatsu** [comment]
Yeah, maybe I'm installing the VM weird? I mean its not a hard task, im setting static ip, dns, gateway and subnet.

**Comment by Mokatsu** [comment]
this is the guide I used last time with kvm: https://infosecspeakeasy.org/t/howto-build-a-cuckoo-sandbox/27

**Comment by doomedraven** [comment]
i hope you used my guide to install kvm-qemu https://github.com/doomedraven/Tools/blob/master/Virtualization/kvm-qemu.sh

glad you solved it :)

**Comment by Mokatsu** [comment]
I did not but this page will definitely be saved for the future, thanks!

**Comment by doomedraven** [comment]
You are welcome ;)
